jose.py

```diff
@@ -1,49 +1,31 @@
+import binascii
+import datetime
+import json
 import logging
-logger = logging.getLogger(__name__)
-
-try:
-    from cjson import encode as json_encode, decode as json_decode
-except ImportError:  # pragma: nocover
-    logger.warn('cjson not found, falling back to stdlib json')
-    from json import loads as json_decode, dumps as json_encode
-
+import six
```

Comment:
You only seem to be using six.b, so normally I would just pull out six.b and have it as a function locally.